src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -186,12 +191,36 @@ export class ZakenService {
     verdeelGegevens.groepId = groep?.id;
     verdeelGegevens.behandelaarGebruikersnaam = medewerker?.id;
     verdeelGegevens.reden = reden;
+    verdeelGegevens.screenEventResourceId = uuidv4();
+    const TIMEOUT = 60 * 60 * 1000;
```

Comment:
You are absolutely right. I will at the very least try to explain it in a comment. The thing is, this timeout is quite arbitrary. It's just there in case the backend never sends the event that the frontend is listening for. I'll see what I can do.

Reply:
Hope this clears it up a bit :)

Reply:
nice!